src/lib/accessibility/Behaviors/Menu/menuDividerBehavior.ts

```diff
@@ -0,0 +1,19 @@
+import { Accessibility } from '../../types'
+
+/**
+ * @description
+ * Behavior for menu divider
+ *
+ * @specification
+ * Adds role 'presentation' to 'root' component's part.
+ */
+
+const menuDividerBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    root: {
+      role: 'presentation',
+    },
+  },
+})
```

Comment:
A little confused with this, we don't have a behavior for `Divider`. How are they differenced?

Reply:
it's just that we did not get to divider yet. it will be done soon. If it turns out to be the same, we will consolidate.

Reply:
The `MenuDivider` is part of the list (it is `li`) element and I guess as the other elements in the menu has this role.